### PR TITLE
iss3160

### LINKED ISSUE #3160: Add new guidelines template
Adding the new guidelines from this doc: https://docs.google.com/document/d/13ckq6KsRKVF7RWDXDvVSkmS8Q3FF-4feNzitMuUZzw4/edit

### PR DESCRIPTION
**Pull request by wolfeallison**
closes #3160

### CODE REVIEW

File: packages/v4/patternfly-docs/content/design-guidelines/content/writing-design-documentation.md [hunk 1]
```diff
@@ -3,26 +3,52 @@ id: Writing design guidelines for PatternFly
...
 
-## Content architecture
+## Design guidelines template
+When creating guidelines, information must be presented in the following order. Sections may be omitted, but if present must be under one of the following headings.
 
-Use headings, hierarchy, formatting, and lists to organize your design guidelines into digestible sections.
+### Elements (H2)
+Introduce the component and list the primary parts that make up its structure. This includes specific icons, headers, fields, symbols, or anything else included in the component.
+Example:
+<img src="sliderelements.png" alt="A screenshot of the slider component's element section with 8 elements outlined." width="479px"/>
```
Comment (mmenestr) [inline packages/v4/patternfly-docs/content/design-guidelines/content/writing-design-documentation.md:14]: I think there's a problem with this img insert that is causing the build to fail, but I'm not 100% sure. Can you make sure you added the img to the right file in the folder structure, and can you try adding a space at the end before the />

File: packages/v4/patternfly-docs/content/design-guidelines/content/writing-design-documentation.md [hunk 2]
```diff
@@ -3,26 +3,52 @@ id: Writing design guidelines for PatternFly
...
 
-## Content architecture
+## Design guidelines template
+When creating guidelines, information must be presented in the following order. Sections may be omitted, but if present must be under one of the following headings.
 
-Use headings, hierarchy, formatting, and lists to organize your design guidelines into digestible sections.
+### Elements (H2)
+Introduce the component and list the primary parts that make up its structure. This includes specific icons, headers, fields, symbols, or anything else included in the component.
+Example:
+<img src="sliderelements.png" alt="A screenshot of the slider component's element section with 8 elements outlined." width="479px" />
```
Comment (evwilkin) [inline packages/v4/patternfly-docs/content/design-guidelines/content/writing-design-documentation.md:14]: This image is inside of another folder, so the path to the image will need to be updated
```suggestion
<img src="./img/sliderelements.png" alt="A screenshot of the slider component's element section with 8 elements outlined." width="479px" />
```

**Review by mmenestr — COMMENTED** [review]
This looks good just have one comment which is to change the "When to use" "When not to sue" and "best practices" into headers that match the template!

File: packages/v4/patternfly-docs/content/design-guidelines/content/writing-design-documentation.md [hunk 3]
```diff
@@ -3,26 +3,53 @@ id: Writing design guidelines for PatternFly
...
+Showcase the different content that could be included within your component. Here is where you can also give additional tips and tricks the user may want to know. Content guidance includes:
+
+- Editorial guidelines for labeling and message text.
+- Length restrictions and what to do when text overflows.
+- Localization considerations.
+
+### Accessibility (H2)
+Each component should list the following sentence:
+
+For information regarding accessibility, visit the [component name] accessibility tab [linked].
```
Comment (mmenestr) [inline packages/v4/patternfly-docs/content/design-guidelines/content/writing-design-documentation.md:49]: This is very confusing - I don't think i'd understand what to link looking at this. Worth adding a sentence to specify which part of the text should be linked in all this

File: packages/v4/patternfly-docs/content/design-guidelines/content/writing-design-documentation.md [hunk 4]
```diff
@@ -101,34 +128,3 @@ If you're linking to other PatternFly pages, use relative URLs instead of full U
 Relative URL: /components/tooltip/design-guidelines
 Full URL: https://www.patternfly.org/v4/components/tooltip/design-guidelines
```
Comment (mmenestr) [inline packages/v4/patternfly-docs/content/design-guidelines/content/writing-design-documentation.md:129]: Need to add an extra empty line between these two lines cause they're showing up on the same line right now. Also would bold "Relative URL:" and "Full URL:"

image

**Review by mmenestr — APPROVED** [review]
Looks good! merging

**Pull request merged**